esphome/components/esp32_ble_server/ble_characteristic.h

```diff
@@ -71,7 +75,7 @@ class BLECharacteristic {
 
  protected:
   bool write_event_{false};
-  BLEService *service_;
+  BLEService *service_ = nullptr;
```

Comment:
```suggestion
  BLEService *service_{};
```
Same result, more modern idiom, keeps code style consistent.